requirements.txt

```diff
@@ -39,5 +39,6 @@ gunicorn
 pytz
 sphinx
 sphinx_rtd_theme
+mock
```

Comment:
@MFAnderson This should not be necessary, is it? mock is installed by tox for the tests, and is not necessary to run the application.